package app54;

public class M38 {
	public static void main(String[] args) {
//		catch(ArithmeticException ex) {
//			
//		}
	}
}
//we can not develop the catch without the try it give the compilation error
//it is not possible